msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497044.java

```diff
@@ -45,8 +46,9 @@
 
 // Interactive auth w/ force_login w/ MFA
 // https://identitydivision.visualstudio.com/DefaultCollection/DevEx/_workitems/edit/497044
-@RetryOnFailure(retryCount = 3) // Seems like AutoMFA does not work sometimes, seems rare but adding extra retries
+@RetryOnFailure(retryCount = 2)
 @RunOnAPI29Minus("Verify Your Identity (MFA)")
+@DoNotRunOnPipeline("Looks like AutoMfa From lab account is still very inconsistent, don't run this on pipeline")
```

Comment:
Have we checked with Lab team about this?

Reply:
I have reached out to Ryan already, but I think he is oof for holidays. Will have to wait until he comes back